config/pipelines/high_throughput_lcm_triomics.wip.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Pipeline configuration for EMSeq
+LCM Triomics EMSeq:
+  pipeline_group: LCM Triomics WGS and EMSeq
+  filters:
+    request_type_key: limber_lcm_triomics_emseq
+    library_type: LCM Triomics EMSeq
+  library_pass: LCMT EM PCRXP
```

Comment:
PCR XP (with space) or PCR-XP (with hyphen) rather than one word as they are 2 different processes (amplification then cleanup)

Reply:
I have copied the purpose name from the process map document and there was no space between **PCR** and **XP**. Now I have asked this in Slack. 

I will update this name and similarly the tube purposes for parts **Pool** and **XP** according to the response.

Reply:
I have received a response for the purpose name: 

_The missing space between PCR & XP for the ‘LCMT EM PCRXP’ is indeed a typo, so please do make the Limber plate name ‘LCMT EM PCR XP’. Re. the pool tube names, those changes look fine._